src/app/tabcontent/tabcontent.component.html

```diff
@@ -0,0 +1,9 @@
+<div>
```

Comment:
Please remove `tabcontent` component from this commit. They will be added later.